mods/ra/maps/allies-13/allies13.lua

```diff
@@ -0,0 +1,301 @@
+RiflePatrolA = { RiflePatrolA1, RiflePatrolA2 }
+RiflePatrolBPath = { PatrolPath8.Location, Generator4.Location, PatrolPath8.Location, CorePathB.Location }
+RiflePatrolB = { RiflePatrolB1, RiflePatrolB2 }
+RiflePatrolCPath = { PatrolPath9.Location, PatrolPath10.Location, CorePathD.Location, PatrolPath11.Location }
+RiflePatrolC = { RiflePatrolC1, RiflePatrolC2 }
+Generators = { Generator1, Generator2, Generator3, Generator4, Generator5, Generator6, Generator7, Generator8 }
+ChargePlaced = { Charge1Placed, Charge2Placed, Charge3Placed, Charge4Placed, Charge5Placed, Charge6Placed, Charge7Placed, Charge8Placed }
+FlameConsoles = { FlameConsole1, FlameConsole2, FlameConsole3, FlameConsole4, FlameConsole5, FlameConsole6, FlameConsole7, FlameConsole8 }
+FlameTowers = { FlameTower1, FlameTower2, FlameTower3, FlameTower4, FlameTower5, FlameTower6, FlameTower7, FlameTower8 }
+DualTowerActors = { DualTower1, DualTower2, DoomBarrel }
+DoomFootprint = { CPos.New(75,77), CPos.New(76,77), CPos.New(77,77), CPos.New(78,77), CPos.New(79,77), CPos.New(80,77), CPos.New(81,77) }
+DoomPatrol = { DoomGren1, DoomGren2, DoomGren3, DoomGren4, DoomGren5, DoomRifle1, DoomRifle2, DoomRifle3, DoomRifle4, DoomRifle5, DoomFlamer1, DoomFlamer2, DoomFlamer3, DoomFlamer4, DoomFlamer5 }
+V2s = { V21, V22, V23, V24, V25, V26, V27, V28 }
+
+TimerLength =
+{
+	easy = DateTime.Minutes(0),
+	normal = DateTime.Minutes(5),
+	hard = DateTime.Minutes(10)
+}
+
+MissionTriggers = function()
+	Trigger.OnAllKilled(Engineers, function()
+		USSR.MarkCompletedObjective(StopAllies)
+	end)
+
+	Trigger.OnAllKilled(V2s, function()
+		Reinforcements.Reinforce(Greece, { "e1", "e1", "e1", "medi" }, { ReinforcementsSouth.Location, SouthTeamStop.Location }, 0)
+		Media.PlaySpeechNotification(Greece, "ReinforcementsArrived")
+	end)
+
+	local doomTriggered
+	Trigger.OnEnteredFootprint(DoomFootprint, function(actor, id)
+		if actor.Owner == Greece and not doomTriggered then
+			Trigger.RemoveFootprintTrigger(id)
+			doomTriggered = true
+
+			Utils.Do(DoomPatrol, function(actor)
+				if not actor.IsDead then
+					IdleHunt(actor)
+				end
+			end)
+		end
+	end)
+
+	Trigger.OnEnteredProximityTrigger(MoneyCrates.CenterPosition, WDist.FromCells(1), function(actor, id)
+		if actor.Owner == Greece then
+			Trigger.RemoveProximityTrigger(id)
+			Greece.MarkCompletedObjective(TakeMoney)
+		end
+	end)
+
+	local flamers = Utils.Where(Map.ActorsInWorld, function(self) return self.Owner == USSR and self.Type == "ftur" end)
+	Utils.Do(flamers, function(actor)
+		Trigger.OnDamaged(actor, function(building)
+			if building.Health < building.MaxHealth * 9/10 then
+				building.StartBuildingRepairs()
+			end
+		end)
+	end)
+
+	Trigger.OnTimerExpired(function()
+		DateTime.TimeLimit = 0
+		Trigger.AfterDelay(1, function() UserInterface.SetMissionText("We're too late!", USSR.Color) end)
+		USSR.MarkCompletedObjective(HoldOut)
```

Comment:
```suggestion
		USSR.MarkCompletedObjective(StopAllies)
```
HoldOut is never defined, and marking it will cause a crash. Swapping it for StopAllies should do.